x/stablestake/types/params.go

```diff
@@ -1,34 +1,53 @@
 package types
 
 import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"gopkg.in/yaml.v2"
 )
 
 var _ paramtypes.ParamSet = (*Params)(nil)
 
+// Parameter keys
+var (
+	KeyDepositDenom   = []byte("DepositDenom")
+	KeyRedemptionRate = []byte("RedemptionRate")
+)
+
 // ParamKeyTable the param key table for launch module
 func ParamKeyTable() paramtypes.KeyTable {
 	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
 }
 
 // NewParams creates a new Params instance
-func NewParams() Params {
-	return Params{}
+func NewParams(depositDenom string, redemptionRate sdk.Dec) Params {
+	return Params{
+		DepositDenom:   depositDenom,
+		RedemptionRate: redemptionRate,
+	}
 }
 
 // DefaultParams returns a default set of parameters
 func DefaultParams() Params {
-	return NewParams()
+	return NewParams("uusdc", sdk.OneDec())
```

Comment:
@jelysn we can find **uusdc** denom in parameter module.

Reply:
Here, there should be possibility of staking non-USDC tokens as well in the future

Reply:
Or multiple stable denoms